src/researches/buildKeywordForms.js

```diff
@@ -41,67 +42,97 @@ const getVariationsApostropheInArray = function( forms ) {
 	} ) ).filter( Boolean );
 };
 
+const filterFunctionWords = function( array, language ) {
+	if ( isUndefined( language ) || language === "" ) {
+		language = "en";
+	}
+
+	const functionWords = getFunctionWords[ language ];
+
+	if ( array.length > 1 && ! ( isUndefined( functionWords ) ) ) {
+		const arrayFiltered = filter( array, function( word ) {
+			return ( ! includes( functionWords.all, word.trim().toLocaleLowerCase() ) );
+		} );
+
+		if ( arrayFiltered.length > 0 ) {
+			return arrayFiltered;
+		}
+	}
+
+	return array;
+};
+
 /**
  * Analyzes the focus keyword string. Checks if morphology is requested or if the user wants to match exact string.
  * If morphology is required the module builds all word forms for all content words (prepositions, articles, conjunctions).
```

Comment:
Not related to this PR, but I think it should be: `all content words (nouns, verbs, adjectives).`